helix-core/src/main/java/org/apache/helix/manager/zk/ZKUtil.java

```diff
@@ -132,7 +139,12 @@ public static boolean isInstanceSetup(String zkAddress, String clusterName, Stri
     return result;
   }
 
-  public static boolean isInstanceSetup(HelixZkClient zkclient, String clusterName,
+  public static boolean isInstanceSetup(HelixZkClient zkClient, String clusterName,
```

Comment:
I think we can move this old API as HelixZkClient can automatically fit into RealmAwareZkClient.

Reply:
I agree. Let me remove old signatures.